target/linux/qualcommax/image/ipq60xx.mk

```diff
@@ -45,6 +45,17 @@ define Device/linksys_mr7350
 endef
 TARGET_DEVICES += linksys_mr7350
 
+define Device/linksys_mr7500
+	$(call Device/linksys_mr)
+	DEVICE_MODEL := MR7500
+	SOC := ipq6018
+	NAND_SIZE := 512m
+	IMAGE_SIZE := 147456k
+	DEVICE_PACKAGES := ipq-wifi-linksys_mr7500 \
+		ath11k-firmware-qcn9074 kmod-ath11k-pci kmod-bluetooth
```

Comment:
Is there a point in including Bluetooth kmod since it most likely does not work at all